docs/app/globals.css

```diff
@@ -1,3 +1,4 @@
+@import "./themes.css";
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
```

Comment:
Use this format please 

```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";
```
Also, add `@import "./themes.css";` at the end.

Reply:
Could you please explain why?